src/sentry/incidents/models/alert_rule.py

```diff
@@ -468,17 +467,25 @@ class Meta:
         db_table = "sentry_alertruletriggeraction"
 
     @property
-    def target(self) -> RpcUser | Team | str | None:
+    def target(self) -> OrganizationMember | Team | str | None:
         if self.target_identifier is None:
             return None
 
         if self.target_type == self.TargetType.USER.value:
-            return user_service.get_user(user_id=int(self.target_identifier))
+            try:
+                return OrganizationMember.objects.get(
+                    user_id=int(self.target_identifier),
+                    organization=self.alert_rule_trigger.alert_rule.organization.id,
```

Comment:
`self.alert_rule_trigger.alert_rule.organization_id` saves you a query to `organization`